contrib/ansible/dashboard/inventory

```diff
@@ -5,13 +5,13 @@ user-server
 [servers:vars]
 results_url="http://{{ inventory_hostname }}"
 run_index="run.example"
+graphql_url="http://{{ inventory_hostname }}:4000"
```

Comment:
@portante Why do we specify a hardcoded port of `4000` here?

Reply:
It is because this is just an example, but I'll take it out.